src/main/resources/static/admin/AdminPanelPage.html

```diff
@@ -0,0 +1,329 @@
+<!doctype html>
+<html lang="en">
+
+<head>
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'favicon' declaration in this 'header' tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)